# CEGUI does not build the .pc files when targetting Windows.
# See upstream ticket: https://bitbucket.org/cegui/cegui/issues/1135
--- old/CMakeLists.txt	2016-04-28 21:12:57.000000000 +0200
+++ new/CMakeLists.txt	2016-07-29 13:07:15.095432616 +0200
@@ -599,7 +599,7 @@
 configure_file( samples/common/include/CEGUISamplesConfig.h.in samples/common/include/CEGUISamplesConfig.h )
 configure_file( doc/doxygen/doxyfile.in doc/doxygen/doxyfile )
 
-if (UNIX AND NOT APPLE)
+if (NOT(APPLE))
     configure_file( cegui/CEGUI.pc.in cegui/CEGUI-${CEGUI_VERSION_MAJOR}.pc @ONLY )
     install(FILES ${PROJECT_BINARY_DIR}/cegui/CEGUI-${CEGUI_VERSION_MAJOR}.pc DESTINATION ${CEGUI_PKGCONFIG_INSTALL_DIR})
 
